source/documentation/other-topics/TZ-Cronjobs.md

```diff
@@ -0,0 +1,53 @@
+### Time Zone Cronjobs
+
+#### Overview
+
+There may be a requirement for some Cronjobs to run at specific times of a day in the UK (including during BST), but Kubernetes Cronjob does not support time zone, our cluster uses UTC timezone internally, and we have no plans to change this. To enable service teams to run cronjobs at specific times, we deployed [TZ Cronjobber][hiddeco-cronjobber](cronjob controller for Kubernetes with support for time zones).
+
+#### Usage
+
+Instead of creating a CronJob like you normally would, you create a TZCronJob, which works exactly the same but supports an additional field: `.spec.timezone`. Set this to the time zone you wish to schedule your jobs in and Cronjobber will take care of the rest.
```

Comment:
Please make "time zone" a link to https://en.wikipedia.org/wiki/List_of_tz_database_time_zones or some other source for the values the user can use.

Reply:
Updated to make a link